src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String LINE = "____________________________________________________________";
+    public static final int MAX = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userCommand; //store user input
+        Task[] taskList = new Task[MAX]; //store all the task from user input
+        int listSize = 0;
+
+        printIntroduction();
+
+        Scanner in = new Scanner(System.in);
+        userCommand = in.nextLine(); //user input
+
+        while (!userCommand.equals("bye")) { //exit command is bye
+            if (userCommand.equals("list")) { //prints list of task
+                printList(taskList, listSize);
+            }
+            else if (userCommand.contains("done ")) { //check user input to see if task completed
+                printCompletedTask(userCommand, taskList);
+            }
+            else {
+                addTask(userCommand, taskList, listSize);
+                printAddedTask(taskList[listSize], listSize);
+                listSize++; //updates the size of list
+            }
+            userCommand = in.nextLine();
+        }
+        printBye();
+    }
```

Comment:
Maybe you can use Switch Cases here to organize the code better and make it more modular for future add-ons.